package.json

```diff
@@ -18,7 +18,10 @@
   "dependencies": {
     "@sap-cloud-sdk/connectivity": "^3.5.0",
     "@sap-cloud-sdk/http-client": "^3.5.0",
-    "@sap-cloud-sdk/util": "^3.5.0"
+    "@sap-cloud-sdk/util": "^3.5.0",
+    "@sap/xssec": "^3.6.0",
```

Comment:
why are these added?

Reply:
I think you should update the `sap-cloud-sdk` libraries. The dependabot alerts might be coming because one of the `sap-cloud-sdk` library uses these  `sap/xssec`, `axios` and `follow-redirects`.

Reply:
done